DRAFT.md

```diff
@@ -904,9 +911,9 @@ constexpr indirect& operator=(indirect&& other) noexcept(
   Otherwise , if `alloc != other.alloc` or `this` is valueless, a new owned
   object is constructed in `this` using
   `allocator_traits<allocator_type>::construct` with the owned object from
-  `other` as the argument as an rvalue, with memory allocated using either the
-  allocator in `this` or the allocator in `other` if the allocator needs
-  updating. The previous owned object in this, if any, is destroyed using
+  `other` as the argument as an rvalue, using either the allocator in `this` or
+  the allocator in `other` if the allocator needs updating. The previous owned
+  object in this, if any, is destroyed using
```

Comment:
```suggestion
  object in `*this`, if any, is destroyed using
```

Reply:
I think most mentions of `this` should be `*this`, since the former is just a pointer and so doesn't own anything.